src/Nav.jsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom"
+
+export const Nav = () => {
+    const username = 'mallionaire'
+    return (
+        <nav>
+            <ul className="nav">
+                <li> Categories: </li>
+                <Link to={`/reviews?category=strategy`}><li> Strategy </li></Link>
```

Comment:
Hi just some advice for when you get to implementing these routes in a later ticket. We don't need to hard code each category option, if we do we will run into issues later if a new category ever gets added, plus it's very DRY code. We have parametric endpoints to help us in this situation - drop me a message if you'd like a recap on that :)